src/common.js

```diff
@@ -18,6 +18,13 @@ export const constants = {
 }
 Object.freeze(constants);
 
+export const executionModes = {
+    CONSENSUS: 0,
+    FALLBACK: 1,
+    READ_REQUEST: 2
+}
+Object.freeze(executionModes);
```

Comment:
We don't need an enum and switch blocks. We could just keep an array of handlers keyed by mode strings. Simply the relevant handler can be executed by doing a key lookup by "mode" string passed in the stdin arg json.